### ISSUE TITLE
Workspace config no longer getting applied correctly in WSL mode

### ISSUE THREAD

**Issue by Stanzilla** [microsoft/vscode-remote-release#3190]
When using the WSL Remote extension to connect to a workspace, certain workbench color customizations no longer work. This worked in 1.45. It also works when not using WSL mode.

Version: 1.46.0 (system setup)
Commit: a5d1cc28bb5da32ec67e86cc50f84c67cc690321
Date: 2020-06-10T09:03:20.462Z
Electron: 7.3.1
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.19041

Steps to Reproduce:
1. Open your project inside of a WSLv2 container using the WSL extension
1. Have a workspace config using these values
```
"workbench.colorCustomizations": {
        "activityBar.background": "#3d9468",
        "activityBar.activeBackground": "#3d9468",
        "activityBar.activeBorder": "#cfbbe3",
        "activityBar.foreground": "#e7e7e7",
        "activityBar.inactiveForeground": "#e7e7e799",
        "activityBarBadge.background": "#cfbbe3",
        "activityBarBadge.foreground": "#15202b",
        "titleBar.activeBackground": "#2e704f",
        "titleBar.inactiveBackground": "#2e704f99",
        "titleBar.activeForeground": "#e7e7e7",
        "titleBar.inactiveForeground": "#e7e7e799",
        "statusBar.background": "#2e704f",
        "statusBarItem.hoverBackground": "#3d9468",
        "statusBar.foreground": "#e7e7e7",
        "tab.activeBorder": "#3d9468",
        "statusBar.border": "#2e704f",
        "titleBar.border": "#2e704f"
    },
```
2. Reload window

-> The titleBar colors don't get applied, the statusBar ones do.

Does this issue occur when all extensions are disabled?: Yes/No
No (can't test since WSL requires extension)

**Comment by bcxpro** [comment]
I am experiencing a similar problem with 1.46 and SSH. In my remote workspaces I use a different color theme so when I open multiple code remotes at the same time I know in which one I am working.

So I have a .vscode/settings.json containing:
```{

    "workbench.colorTheme": "Abyss"
}```

This worked until 1.45 fine but it just stopped working in 1.46. I had to roll back to 1.45.

**Comment by Stanzilla** [comment]
Extra info: If you edit the file and save, the colors get applied fine, just not on first load.

**Comment by Tyriar** [comment]
FYI @sbatten

**Comment by Stanzilla** [comment]
So I think this is fixed, just not sure if it was 1.47 or the Remote Containers extension update

**Comment by bcxpro** [comment]
It still fails for me in 1.47 with SSH.

**Comment by Stanzilla** [comment]
Okay reopening then

**Comment by bcxpro** [comment]
@Stanzilla I was testing today with 1.47.2 and your example and it seems to work both on SSH and WSL. However my much simpler case which consist of a single theme statemen does not work in either mode. Both worked OK until 1.45 and your case seems to be working back on 1.47.
My case is jus a .vscode/settings.json file containing:
`{
	    "workbench.colorTheme": "Abyss"
}`
It always open a default theme both in SSH and WSL. If I edit the file while the workspace is open and save with a different theme, then it applies the theme (until I close it and open again)

**Comment by anthonyalayo** [comment]
+1, I am struggling with this as well.

**Comment by bcxpro** [comment]
It seems to be fixed in today's insider's build . Closed on #101893

Version: 1.52.0-insider (user setup)
Commit: 24b28f57be22fe3029cb17a1dd72d8d9c2d6468b
Date: 2020-11-06T05:43:01.017Z
Electron: 9.3.3
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.19042

**Comment by aeschli** [comment]
Fixed by https://github.com/microsoft/vscode/issues/101893
